draft-irtf-qirg-principles-03.xml

```diff
@@ -795,6 +820,47 @@
            information via separate classical channels which the repeaters will
            have to correlate with the qubits stored in their memory.</t>
 
+           <t> The first quantum networks may not be "routing and forwarding"
+           networks.
+           <vspace blankLines="1" />
+
+           Quantum links provide Bell pairs that are undirected network
+           resources, in contrast to directed links of classical networks.
+           This phenomenological distinction should result in architectural
+           differences between quantum networks and classical networks.
+           <vspace blankLines="1" />
+
+           Classical networks have routing and forwarding; routing decides
+           the way to go, and forwarding executes actual data transfer,
+           obeying the decision. Quantum networks perform routing in a control
+           plane and execute entanglement swapping instead of forwarding in
+           a data plane. Such quantum networks are "routing and swapping" networks.
+           In "routing and swapping" networks, we do not need to care about
+           the order of generating link Bell pairs since Bell pairs are
+           undirected resources. This distinction makes control algorithms and
+           optimization of quantum networks different from classical ones.
+           Additionally, "routing and swapping" quantum networks are not
+           "store and forward" networks.
+           <vspace blankLines="1" />
+
```

Comment:
The first sentence is too detailed in technical details. Since entanglement swapping is already explained in this document, why not refer to it?

Reply:
I agree.